measurements/app.py

```diff
@@ -94,6 +93,16 @@ def init_app(app, testmode=False):
 def check_config(config):
     pass
 
+def extract_client_ipaddr_for_throttling():
+    ipaddr = flask_limiter.util.get_ipaddr()
+    # TODO: move to an external config file /etc/ooniapi.conf ?
+    explorer_addr = "37.218.245.0"
```

Comment:
Beware that the last octet is getting sanitized by our nginx config, so the IP of the explorer likely has an extra octet in here.

Reply:
The last .0 octet is due to that.